Docs: Add note about paid plugins to homepage (#1392)
grafana/plugin-tools: docusaurus/docs/get-started/get-started.mdx

Grafana plugin development allows you to create many different types of user experiences. For example, you can make:

- **Panel plugins** - new ways of visualizing data
- **Data source plugins** - connections to a new database or other source of data
- **App plugins** - integrated out-of-the-box experiences

:::tip

If this is your first time creating a plugin, we recommend that you familiarize yourself with the fundamentals of plugin types, backend plugins, data frames, and other essentials. Learn more about the [key concepts of Grafana plugin development](/key-concepts/).

:::

## Classifications of Grafana plugins

> We recommend that you familiarize yourself with the signature classifications of Grafana plugins, such as the distinction between private and public plugins. Note that if you are a commercial entity wanting to publish to the official Grafana catalog, a paid subscription is typically required. Learn more about [our plugins policy](https://grafana.com/legal/plugins/).

Reviewer: ```suggestion
We recommend that you familiarize yourself with the signature classifications of Grafana plugins, such as the distinction between private and public plugins. Note that if you want to publish a plugin which is associated with a commercial offering to the official Grafana catalog, a paid subscription is typically required. Learn more about [our plugins policy](https://grafana.com/legal/plugins/).
```
